documentation/modules/exploit/linux/http/terramaster_unauth_rce_cve_2021_45837.md

```diff
@@ -0,0 +1,83 @@
+## Vulnerable Application
+
+This module provides a Terramaster chained exploit that performs session crafting to achieve escalated privileges
+that allows an attacker to access vulnerable code execution flaws. TOS versions `4.2.15` and below are affected.
```

Comment:
What is `TOS`?

Reply:
TOS stands for TerraMaster Operating System (TOS). It is basically a "hardened Linux OS". I will spell it out in the document to make it more clear for the reader.